platform/broadcom/platform-modules-accton.mk

```diff
@@ -0,0 +1,14 @@
+# Accton Platform modules
```

Comment:
Is it better to have your own repository to put the platform modules? So that your platform modules code update could be independent to our repository. Similar to other platform modules.